cumulus_library_opioid/vocab/vsac.py

```diff
@@ -68,27 +69,26 @@ def download_oid_data(
     return True
 
 
-def main(cli_args=None):
-    """Temporary CLI interface"""
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "source_vocab", help="Name of vocab to look up codes for", default=None
-    )
-    parser.add_argument("--api_key", help="UMLS api key", default=None)
-    parser.add_argument(
-        "--force-recreate",
-        help="Force redownloading of data even if it already exists",
-        action="store_true",
-    )
-    parser.add_argument("--path", help="optional path to write data to", default=None)
-    args = parser.parse_args(cli_args)
-    return download_oid_data(
-        args.source_vocab,
-        api_key=args.api_key,
-        force_recreate=args.force_recreate,
-        path=pathlib.Path(args.path),
-    )
-
-
-if __name__ == "__main__":
-    main()
+def get_vsac_stewards(config:base_utils.StudyConfig) -> list[str]:
+    """Gets the steward target(s) from the StudyConfig options"""
+    stewards = []
+    try:
+        if config.options['steward'] == 'all':
+            for k,v in VSAC_OIDS.items():
+                if v is not None:
+                    stewards.append(k)
+        else:
+            if config.options['steward'] in VSAC_OIDS.keys():
+                stewards.append(config.options['steward'])
+            else:
+                sys.exit(
+                    f"{config.options['steward']} is not a valid value set steward."
+                    f" \n\n{VALID_MSG}"
+                )
+        return stewards
+    except TypeError:
+        sys.exit(
+            "This study requires specifying a value set steward. Add an argument like"
+            "'--option steward:name' to specify which value sets to use.\n\n"
+            f"{VALID_MSG}"
+        )
```

Comment:
How did `--option` feel here? I still haven't adopted that in data-metrics (hmm, I should go do that). Curious how the pattern feels in real world uses. Looks fine from here.

Reply:
seemed pretty reasonable from this end, and i think the exit message gets to expected usage, so no immediate bad feelings.